driver_integration_test.go

```diff
@@ -123,3 +127,34 @@ func TestDriverOpenDefaultEngine(t *testing.T) {
 func TestDriverExecStatement(t *testing.T) {
 	runTestDriverExecStatement(t, dsnMock)
 }
+
+// TestDriverSystemEngine checks system engine queries are executed without error
+func TestDriverSystemEngine(t *testing.T) {
+	suffix := strings.ReplaceAll(uuid.New().String(), "-", "")
+	databaseName := fmt.Sprintf("gosdk_system_engine_test_%s", suffix)
+	engineName := fmt.Sprintf("gosdk_system_engine_test_e_%s", suffix)
+	engineNewName := fmt.Sprintf("gosdk_system_engine_test_e_2_%s", suffix)
+
+	db, err := sql.Open("firebolt", dsnSystemEngineMock)
+	if err != nil {
+		t.Errorf("failed unexpectedly with %v", err)
+	}
+	queries := []string{
+		fmt.Sprintf("SHOW DATABASES"),
+		fmt.Sprintf("SHOW ENGINES"),
+		fmt.Sprintf("CREATE DATABASE %s", databaseName),
+		fmt.Sprintf("CREATE ENGINE %s WITH SPEC = 'C1' SCALE = 1", engineName),
+		fmt.Sprintf("ATTACH ENGINE %s TO %s", engineName, databaseName),
+		fmt.Sprintf("ALTER DATABASE %s WITH DESCRIPTION = 'GO SDK Integration test'", databaseName),
+		fmt.Sprintf("ALTER ENGINE %s RENAME TO %s", engineName, engineNewName),
+		fmt.Sprintf("START ENGINE %s", engineNewName),
+		fmt.Sprintf("STOP ENGINE %s", engineNewName),
+		fmt.Sprintf("DROP DATABASE %s", databaseName)}
+
+	for _, query := range queries {
+		_, err := db.Query(query)
```

Comment:
Can we also try to read result here is any? So we also test the we support the output?